src/main/scala/outwatch/dom/DomTypes.scala

```diff
@@ -17,6 +17,12 @@ import monix.reactive.OverflowStrategy.Unbounded
 
 import scala.scalajs.js
 
+private[outwatch] object Builders {
+  type Attribute[T, _] = helpers.ValueBuilder[T, Attr]
```

Comment:
maybe can also live in `DomTypesBuilders`?

Reply:
I'm fine with both. We'll see how it turns out in the next refactoring 😉